### DISCUSSION TITLE
How can I turn off the Alt-C key binding?

### DISCUSSION THREAD

**Discussion by hjudt** [junegunn/fzf#2893] [Q&A] [answered]
I'm using bash and I am very used to capitalizing a word with Alt-C, and when using fzf it is very unexpected for me to see it go into action when I hit this key combo. How can I make fzf not override this keybinding, or make it bind something else?

**Comment by junegunn** [comment, marked as answer]
You can restore the original bash bindings like so

```sh
[ -f ~/.fzf.bash ] && source ~/.fzf.bash

if [[ $- =~ i ]]; then
  # Restore original bindings
  # bind '"\C-t": transpose-chars'
  # bind '"\C-r": reverse-search-history'
  bind '"\ec": capitalize-word'
fi
```

**Comment by hjudt** [comment]
Thank you for the snippet, this worked.
